webapp/src/main/webapp/templates/freemarker/visualization/personlevel/coAuthorPersonLevelD3.ftl

```diff
@@ -39,16 +39,16 @@ var visualizationDataRoot = "${dataVisualizationURLRoot}";
 // -->
 
 var i18nStringsCoauthorship = {
-    coAuthorsString: '${i18n().co_authors_capitalized}',
-    authorString: '${i18n().author_capitalized}',
-    publicationsWith: '${i18n().publications_with}',
-    publicationsString: "${i18n().through_today}",
-    coauthorsString: '${i18n().co_author_s_capitalized}'
+    coAuthorsString: '${i18n().co_authors_capitalized?js_string}',
+    authorString: '${i18n().author_capitalized?js_string}',
+    publicationsWith: '${i18n().publications_with?js_string}',
+    publicationsString: '${i18n().through_today?js_string?js_string}',
```

Comment:
Looks like you doubled the fun on this one.

Reply:
in germany we call it "doppelt hält besser" it is like "twice as strong", but yes it was a typo, i removed it